### PR TITLE
Remove add-delete-to-pr-files feature

### LINKED ISSUE #1458: Double delete icons
I see double delete icons in the diff view. When I disable refined-github I only see one.

afbeelding

### PR DESCRIPTION
**Pull request by vinkla**
This has been implemented by GitHub. Fixes #1458.

### CODE REVIEW

**Comment by sindresorhus** [comment]
@vinkla Can you fix the merge conflict?

**Comment by vinkla** [comment]
@sindresorhus done now.

File: readme.md [hunk 1]
```diff
@@ -190,6 +189,8 @@ And [many more…](source/content.css)
 - ~~[Permalinked comments are highlighted.](https://user-images.githubusercontent.com/1402241/37349492-226bd37a-2709-11e8-8087-d9686b330240.png)~~ - *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-26-highlighting-of-permalinked-comments/)*
 - ~~Leave a single comment in pull request diffs instead of starting a review by pressing <kbd>shift</kbd> <kbd>enter</kbd>.~~ - *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-24-keyboard-shortcut-to-leave-a-single-comment-in-pull-requests-reviews/)*
 
+- ~~[Quickly edit a repository's README from the repository root.](https://user-images.githubusercontent.com/170270/27501200-31a1fa20-586c-11e7-9a3f-ce270014bf0a.png)~~ *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-24-button-to-edit-a-repositorys-readme-from-the-repository-root/)*
+- ~~[Quickly delete a file from pull requests.](https://user-images.githubusercontent.com/1402241/42529637-578587e4-847f-11e8-9705-57384a1edd24.png)~~ - *[Implemented by GitHub](https://github.com/sindresorhus/refined-github/issues/1458)*
```
Comment (sindresorhus) [inline readme.md]: There's no a changelog entry for this: https://blog.github.com/changelog/2018-07-31-removing-files-from-a-pull-request/

File: readme.md [hunk 2]
```diff
@@ -190,6 +189,8 @@ And [many more…](source/content.css)
 - ~~[Permalinked comments are highlighted.](https://user-images.githubusercontent.com/1402241/37349492-226bd37a-2709-11e8-8087-d9686b330240.png)~~ - *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-26-highlighting-of-permalinked-comments/)*
 - ~~Leave a single comment in pull request diffs instead of starting a review by pressing <kbd>shift</kbd> <kbd>enter</kbd>.~~ - *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-24-keyboard-shortcut-to-leave-a-single-comment-in-pull-requests-reviews/)*
 
+- ~~[Quickly edit a repository's README from the repository root.](https://user-images.githubusercontent.com/170270/27501200-31a1fa20-586c-11e7-9a3f-ce270014bf0a.png)~~ *[Implemented by GitHub](https://blog.github.com/changelog/2018-07-24-button-to-edit-a-repositorys-readme-from-the-repository-root/)*
```
Comment (sindresorhus) [inline readme.md]: This should not be here.

Comment (vinkla) [reply readme.md]: Hmm, wasn't this added in https://github.com/sindresorhus/refined-github/commit/a953ed4948e4dd5a73fc94413a1368b4b1c3665a?

Comment (sindresorhus) [reply readme.md]: Yes, but it should not be part of *your* diff here. Notice how it's green.

**Comment by vinkla** [comment]
I've pushed an update. Let me know if there is anything else I should update before we can merge this.

**Comment by sindresorhus** [comment]
See my comment in https://github.com/sindresorhus/refined-github/pull/1462#discussion_r207594552

File: readme.md (hunk 2 above)
Comment (vinkla) [reply readme.md]: Hmm, I'm not sure how I can remove the diff. Any pointers?

Comment (sindresorhus) [reply readme.md]: You just edit it out. You have it now because you resolved the merge conflict incorrectly.

Comment (vinkla) [reply readme.md]: Oh, I see. Sorry about that.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**